src/test/java/uk/gov/register/functional/DataDownloadFunctionalTest.java

```diff
@@ -144,6 +151,17 @@ public void downloadPartialRSF_shouldReturnReturn404ForRsfBoundariesOutOfBound()
         assertThat(response.getStatus(), equalTo(400));
     }
 
+    @Test
+    public void downloadPartialRSF_shouldReturnSameRootHashAsDownloadRSF() {
```

Comment:
that's a really cool test, can we drop just hashes being tested so:
`assertThat(fullRsfLines, equalTo(partialRsfLines));` and rename it `shouldReturenTheSameRSF bla bla bla`?